recipes/taproom/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "taproom" %}
+{% set version = "0.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/slochower/host-guest-benchmarks/archive/v{{ version }}.tar.gz
+  sha256: ac90fae53750cf0ff944d4e940b0398f55bd33f3e3153fe3c3dbfc4200c69dc8
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
```

Comment:
I think you'll need `setuptools` as a `run` dependency since the entry points use `pkg_resources`. This isn't obvious from the import test of `taproom` since it doesn't actually hit the code that uses this.